config/sys.config

```diff
@@ -39,9 +41,10 @@
    },
    {report, [
              {reporters, [
-                          {exometer_report_statsd, []}
+                          {exometer_report_tty, []}
```

Comment:
This shouldn't be switched for placement in the production system. Otherwise you'll kill all the metrics exiting the system. :)